tsconfig.json

```diff
@@ -36,6 +36,7 @@
     ".next/types/**/*.ts"
   ],
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "packages"
```

Comment:
Both Next.js and Shortest use `@` as path alias, which causes a conflict. For that reason, I had to split the typecheck. I think Next.js should be moved to `apps/nextjs` for a better monorepo organization.